Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG ALPINE_VERSION=3.10.1
 ARG GOLANG_VERSION=1.12.9-alpine3.10
 FROM golang:$GOLANG_VERSION as builder
 
-RUN apk add --no-cache bash build-base git python unzip wget
+RUN apt-get update && apt-get install -y bash git unzip wget jq vim
```

Comment:
Does this run? Alpine doesn't have `apt-get` does it?

Reply:
I see, it's being overridden in the Makefile. I'd recommend either making the builder use Alpine or switch the runtime image to a Debian derivative, mixing glibc and musl eventually causes pain. If this is desirable for some reason, let's remove the Alpine default value from `GOLANG_VERSION`